app/_layout.tsx

```diff
@@ -0,0 +1,5 @@
+// import { Stack } from "expo-router";

```

Comment:
What is this for ?

Reply:
_layout.tsx its a layout route that uses the stack component of react navigation. commented it so that the app will not have a header.